clang-tools-extra/test/clang-doc/basic-project.test

```diff
@@ -7,6 +7,7 @@
 // RUN: FileCheck %s -input-file=%t/docs/GlobalNamespace/Rectangle.html -check-prefix=HTML-RECTANGLE
 // RUN: FileCheck %s -input-file=%t/docs/GlobalNamespace/Circle.html -check-prefix=HTML-CIRCLE
 
+// JSON-INDEX: var RootPath = "{{.*}}";
```

Comment:
This is a URL, right? if so, is there a reason it wouldn't match, regardless of platform? I'm a bit surprised this is the only change required to the tests, so I'm wondering if we're doing a good job checking things in our existing HTML tests.

Reply:
This isn't a url, this variable is a path to the directory where clang-doc is built from. I wanted the html to still render properly when viewed via the local filesystem so I added this export variable.
Most the changes that is reflected in this patch can only be seen once the browser loads the page and javascript generates the sidebar with the indexes, I don't think its possible to see the changes reflected unless we incorporate some sort javascript test framework. We could also just generate the indexes via the c++ html generator than we be able to see the changes.
I've added some test that test the different behaviour when we generate clang-doc via a relative path or an absolute one, though

Reply:
Oh, then I'm not 100% sure we want to replace all the `\` w/ `/`. I know you mentioned browser behavior, but I'm not 100% certain that's a great motivation. When I thought it was a URL, there's obviously no issue, but, for a file system path, I'm hesitant to say with confidence that there's no downside.

Also, given the test updates, IDK if this check is particularly useful. I don't see much difference in the check as is, and just 
```
// JSON-INDEX: var RootPath =
```